mozilla-release/dom/ipc/PBrowser.ipdl

```diff
@@ -162,6 +162,8 @@ parent:
     sync DispatchFocusToTopLevelWindow();
 
 parent:
+    async LoadContextPrivatenessChanged(bool isPrivate);
```

Comment:
This is a declaration of IPC message from content process to main process, notifying about DocShell's privacy flag change.